mappings/net/minecraft/block/BeeHiveBlock.mapping

```diff
@@ -1,9 +1,33 @@
-CLASS net/minecraft/class_4481 net/minecraft/block/BeeHiveBlock
+CLASS net/minecraft/class_4481 net/minecraft/block/BeehiveBlock
```

Comment:
The file is still called `BeeHiveBlock.mapping` with a capital H.

Reply:
Grr, blame Windows for that one. It's renamed on my actual computer, but Windows always acts weird with case in file names, so it didn't commit the name change.